helm/defectdojo/templates/celery-beat-deployment.yaml

```diff
@@ -23,11 +23,26 @@ spec:
         app.kubernetes.io/name: {{ include "defectdojo.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
+      serviceAccountName: defectdojo
       {{- if .Values.imagePullSecrets }}
       imagePullSecrets:
       - name: {{ .Values.imagePullSecrets }}
       {{- end }}
       containers:
+      {{- if .Values.cloudsql.enabled  }}
```

Comment:
Is it possible to define this code only once? Maybe including it or define a separate service. Otherwise we have a lot of duplicate code or do I miss some benefits of this solution?

Reply:
I'll check if is possible to have this snippet into an helper and include when is needed.

Reply:
I tried to play with the template to include this snippet but I haven't come to a good solution. Are you fine if we merge this in this form and try to follow up later with a refactoring?

Reply:
If there is no quick solution it is fine for me.